examples/docs/tsconfig.json

```diff
@@ -2,7 +2,8 @@
   "compilerOptions": {
     "target": "es2020",
     "module": "esnext",
-    "jsx": "preserve"
+    "jsx": "preserve",
+    "moduleResolution": "node",
+    "types": ["astro/env"]
```

Comment:
For `import.meta.env` support inside non-Astro files, it's better to use `"types": ["vite/client"]` than `astro/env`. 

Using `astro/env`, makes the Astro global available in non-Astro files, which doesn't work (at the moment)

Reply:
Yup, that's what I had before but then I found the docs which suggested using `astro/env`. https://docs.astro.build/en/guides/typescript/#setup

Changed.